shared/fs/footer/upload.desktop.js

```diff
@@ -37,26 +38,46 @@ const realCSS = `
 }
 `
 
-const Upload = ({showing, files, totalSyncingBytes, timeLeft, debugToggleShow}: UploadProps) => (
-  <React.Fragment>
-    {!!debugToggleShow && <Button type="Primary" onClick={debugToggleShow} label="Toggle" />}
-    <CSSTransition in={showing} classNames="upload-animation" timeout={300} unmountOnExit={true}>
-      <Box className="upload-animation-loop" style={stylesBox}>
-        <style>{realCSS}</style>
-        <Text key="files" type="BodySemibold" style={stylesText}>
-          {files
-            ? `Encrypting and uploading ${files} files...`
-            : totalSyncingBytes
-              ? 'Encrypting and uploading...'
-              : 'Done!'}
-        </Text>
-        {!!(timeLeft && timeLeft.length) && (
-          <Text key="left" type="BodySmall" style={stylesText}>{`${timeLeft} left`}</Text>
-        )}
-      </Box>
-    </CSSTransition>
-  </React.Fragment>
-)
+const styles = Styles.styleSheetCreate({
+  textOverflow: Styles.platformStyles({
+    common: {
+      color: globalColors.white,
+      maxWidth: '60%',
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textAlign: 'center',
+      textOverflow: 'ellipsis',
```

Comment:
This should be in `isElectron`.

Reply:
Let me change

Reply:
Oops nvmind you're right. Maybe the entry should be named something other than the same thing as a css class, to reduce confusion.